src/test/scala/ai/privado/testfixtures/PhpFrontendTestSuite.scala

```diff
@@ -30,4 +31,26 @@ class TestCpgWithPhp(val fileSuffix: String, val language: Language.Value) exten
 }
 
 class PhpFrontendTestSuite(fileSuffix: String = ".php", language: Language.Value = Language.PHP)
-    extends PrivadoBaseTestFixture(() => new TestCpgWithPhp(fileSuffix, language)) {}
+    extends PrivadoBaseTestFixture(() => new TestCpgWithPhp(fileSuffix, language)) {
+
+  protected val collectionRules: List[RuleInfo] = List(
```

Comment:
If the rule is used across, we need to define it in the `rule` package in test.
If it is used in a specific test cases, we can define it there itself.
If it is used in multiple test files, you can create a language package in rule, to store such repeated rules